libass/ass.h

```diff
@@ -24,7 +24,7 @@
 #include <stdarg.h>
 #include "ass_types.h"
 
-#define LIBASS_VERSION 0x01500000
+#define LIBASS_VERSION 0x01500001
```

Comment:
Not sure which commit should have this bump. Previous commit also changed API.

Reply:
If `ass_get_feature_state` remains, both commits should have a bump imho.